spec/models/restitution/avec_entrainement/temps_entrainement_spec.rb

```diff
@@ -4,7 +4,7 @@
 
 describe Restitution::AvecEntrainement::TempsEntrainement do
   let(:metrique_temps_entrainement) do
-    described_class.new(nil, evenements).calcule
+    described_class.new.calcule(nil, evenements)
```

Comment:
On ne voudrait pas avoir un array vide à la place du `nil` ? J'ai bien conscience que ce n'est pas toi qui a introduit ces arguments mais ca pourrait être l'occasion d'harmoniser avec les tests ci-dessous. 😃

Reply:
Ben si, c'est bien moi. C'est fait, j'ai homogénéisé.